Name: No Airplane Mode in Device Menu
Version: 1.3.1-1
Author: Jason Robitaille
Description: Removes the airplane mode toggle from the device menu  (MIT license)

--- .orig/usr/lib/luna/system/luna-systemui/app/controllers/devicemenu-assistant.js
+++ /usr/lib/luna/system/luna-systemui/app/controllers/devicemenu-assistant.js
@@ -64,10 +64,7 @@
 		//Spinner for Airplande Mode
 		//this.controller.setupWidget('apSpinner', this.spinnerAttrs, this.apSpinnerModel);	    	
 			
-		if(this.barAssistant.getAirplaneMode()) 			
-			this.controller.get('dm_airplanemode_status').innerHTML = $L('Turn off Airplane Mode');
-		else
-			this.controller.get('dm_airplanemode_status').innerHTML = $L('Turn on Airplane Mode');
+		this.controller.get('dm_airplanemode').style.display="none";
 			
 		var btstate = this.barAssistant.getCurrentBluetoothState();
 		if(btstate === 'Off') {
